pyro/incompressible_viscous/problems/converge.py

```diff
@@ -0,0 +1,66 @@
+r"""
+Initialize a smooth incompressible+viscous convergence test.  Here, the
+velocities are initialized as
+
+.. math::
+
+    u(x,y) = 1 - 2 \cos(2 \pi x) \sin(2 \pi y)
+
+    v(x,y) = 1 + 2 \sin(2 \pi x) \cos(2 \pi y)
+
+and the exact solution at some later time t, for some viscosity nu, is
+
+.. math::
+
+    u(x,y,t) = 1 - 2 \cos(2 \pi (x - t)) \sin(2 \pi (y - t)) e^{-16 \pi^2 \nu t}
```

Comment:
I tried to test the convergence using by comparing the higher-resolution solutions to lower-resolution solutions and I was able to get ~1.5 order of convergence. So I quickly checked here and these solutions don't seem to work. 
I changed the initial condition and the exact solutions that are described in "I do like CFD", and it seems like it's also converging to 1.5 order now.

Reply:
Can you explain what you did? I tried the problem in the textbook and I still get large errors

Reply:
Following the text, I just changed the initial condition in `converge.py` to:
```
u = -cos(2pi x)sin(2pi y)
v = sin(2pi x)cos(2pi y)
```
then change the corresponding exact solutions in `incomp_viscous_converge.py`:
```
u = -cos(2pi x)sin(2pi y)exp(-8pi^2 nu t)
v = sin(2pi x)cos(2pi y)exp(-8pi^2 nu t)
```

Another trivial thing I did is to add the gradp terms and the diffusion terms before calculating the transverse terms in `get_interface_states()`, I don't think this has a large impact though.